pkg/network/usm/monitor_test.go

```diff
@@ -426,20 +515,21 @@ func allTrue(x []bool) bool {
 		}
 	}
 	return true
+
 }
 
 func testHTTPMonitor(t *testing.T, targetAddr, serverAddr string, numReqs int, o testutil.Options) {
 	monitor := newHTTPMonitor(t)
 
 	srvDoneFn := testutil.HTTPServer(t, serverAddr, o)
+	defer srvDoneFn()
```

Comment:
the original location meant to ensure we're closing all connection before "pulling them from the kernel"
how is that ok to move the closer function to defer?

Reply:
Thanks, I messed up here